python/lsst/ts/MTAirCompressor/enums.py

```diff
@@ -0,0 +1,29 @@
+# This file is part of ts_MTAirCompressor.
+#
+# Developed for the Vera Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["ErrorCode"]
+
+import enum
+
+
+class ErrorCode(enum.IntEnum):
```

Comment:
Please document this enum class.

If this is the enum codes that your CSC can use to to go FAULT state then I strongly suggest this be moved to ts_idl. It may be too late for cycle 26, but at least file a ticket (and possibly add it to ts_idl now, but use a local copy if needed).

Reply:
Documented in https://github.com/lsst-ts/ts_MTAirCompressor/pull/6/commits/493bc20ea2ddd16c8840557a83146d6ac697d468